app/views/kaminari/_paginator.html.haml

```diff
@@ -0,0 +1,12 @@
+= paginator.render do
+  %nav.pagination.flex.items-center.justify-center.space-x-1
+    = first_page_tag unless current_page.first? 
+    = prev_page_tag unless current_page.first?
+    - each_page do |page|
```

Comment:
maybe its better empty line before and after `each`. just to make the code easier to read

Reply:
There is no such empty lines in default [kaminari views](https://github.com/kaminari/kaminari/blob/a399320a063b465b42fea7e79ab2feda2198e155/kaminari-core/app/views/kaminari/_paginator.html.haml), so I think we don't need them either :slightly_smiling_face: